jekyll/_cci2/local-cli.md

```diff
@@ -15,16 +15,20 @@ This document provides instructions for installing and using the CircleCI CLI.
 ## Overview
 {:.no_toc}
 
-The `circleci` commands enable you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase. 
+The `circleci` command-line tool enables you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase.
 
 You can also run `circleci` commands in your [`.circleci/config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file for jobs that use the primary container image. This is particularly useful for globbing or splitting tests among containers.
 
-**Note:** There are currently two versions of the CLI that you may install. The Local CLI available at <https://circle-downloads.s3.amazonaws.com/releases/build_agent_wrapper/circleci> is an older version that does not yet support Workflows.
-The Local CLI version available at <https://github.com/CircleCI-Public/circleci-cli> has a different set of options, some of which are still in active development, and will be the replacement for the old CLI.
+**Note:** An older version of the circleci cli has been replaced by a new version available at <https://github.com/CircleCI-Public/circleci-cli>. See upgrading and installation instructions below.
```

Comment:
We should probably put a date on this. Or say something like "If you installed the CLI prior to October 2018 you may need to do an extra one-time step to switch over to the newer one."